.github/workflows/docs.yml

```
@@ -8,15 +8,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@master
+      with:
+        persist-credentials: false
+        fetch-depth: 0
     - name: Gen Docs
       run: make docker-compose
-    - name: Create Pull Request
-      if: "!contains(github.event.head_commit.message, 'skip ci')"
-      uses: peter-evans/create-pull-request@v2
+    - name: Commit files
+      run: |
+        git config --local user.email "syslbot@anz.com"
+        git config --local user.name "SyslBot"
+        git add .
+        git commit -m "Regenerate docs" && echo ::set-env name=can_push::1 || echo "No changes to docs"
+    - name: Push changes
+      if: env.can_push == 1
+      uses: ad-m/github-push-action@master
```

I might suggest we use a fixed tag version, in case of breaking changes
```suggestion
      uses: ad-m/github-push-action@v0.6
```